app/src/main/java/com/example/gotproject/AppModule.kt

```diff
@@ -0,0 +1,43 @@
+package com.example.gotproject
+
+import android.content.Context
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+class AppModule {
+
+    @Provides
+    @Singleton
+    fun getAppDatabase(context: Context): AppDatabase {
```

Comment:
You're getting this error because Hilt does not know what Context you are referring to. Fix it by specifying the app context:
`fun getAppDatabase(@ApplicationContext context: Context): AppDatabase {`

Reply:
It worked, thanks